docs/x509/tutorial.rst

```diff
@@ -82,3 +82,70 @@ a few details:
     ...     f.write(csr.public_bytes(serialization.Encoding.PEM))
 
 Now we can give our CSR to a CA, who will give a certificate to us in return.
+
+Creating a self-signed certificate
+----------------------------------
+
+While most of the time you want a certificate that has been *signed* by someone
+else (i.e. a certificate authority), so that trust is established, sometimes
+you want to create a self-signed certificate. Self-signed certificates are not
+issued by a certificate authority, but are instead signed by themselves.
```

Comment:
Yeah. Let me try rephrasing.

On Mon, Mar 14, 2016 at 12:30 PM, Hynek Schlawack <notifications@github.com>
wrote:

> In docs/x509/tutorial.rst
> <https://github.com/pyca/cryptography/pull/2827#discussion_r56029729>:
>
> > @@ -82,3 +82,70 @@ a few details:
> >      ...     f.write(csr.public_bytes(serialization.Encoding.PEM))
> >
> >  Now we can give our CSR to a CA, who will give a certificate to us in return.
> > +
> > +Creating a self-signed certificate
> > +----------------------------------
> > +
> > +While most of the time you want a certificate that has been *signed* by someone
> > +else (i.e. a certificate authority), so that trust is established, sometimes
> > +you want to create a self-signed certificate. Self-signed certificates are not
> > +issued by a certificate authority, but are instead signed by themselves.
>
> The certificate doesn’t sign itself. You create a key first that you use
> for signing, no?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pyca/cryptography/pull/2827/files#r56029729>.
>



-- 
"I disapprove of what you say, but I will defend to the death your right to
say it." -- Evelyn Beatrice Hall (summarizing Voltaire)
"The people's good is the highest law." -- Cicero
GPG Key fingerprint: D1B3 ADC0 E023 8CA6